Library/Formula/luvi.rb

```diff
@@ -0,0 +1,19 @@
+class Luvi < Formula
+  desc "A project in-between luv and luvit, lua packages."
+  homepage "https://github.com/luvit/luvi"
+  url "https://github.com/luvit/luvi/releases/download/v2.1.8/luvi-src-v2.1.8.tar.gz"
+  sha256 "1ef449c6a76bcf7a03c8bbcdd5423b0366d63b10c52fd463ca2ba8aaf9248781"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "make", "regular-asm"
+    system "make"
+    bin.install "build/luvi"
+  end
+
+  test do
+    system "make", "test"
```

Comment:
So this should work, but I need to manually copy the test app to the test path as part of the build step? https://github.com/creationix/homebrew/commit/65f3ff283c5b037807260be76b0ed0a2efc6303a#diff-fa25f5d63de4da959fc7c7808ad02f02R16